src/components/vamcFacility/template.test.tsx

```diff
@@ -119,10 +119,10 @@ describe('HealthCareLocalFacility with valid data', () => {
 
     expect(
       screen.getByRole('heading', {
-        name: 'Other services at VA Boston health care',
+        name: /Other services at Framingham VA Clinic/,
       })
     ).toBeInTheDocument()
-    expect(container.querySelectorAll('ul > li > p > va-link')).toHaveLength(8)
+    expect(container.querySelectorAll('ul > li > p > va-link')).toHaveLength(10)
```

Comment:
Note that this has changed, but I may add the limit back depending on what Laura says